src/index.js

```diff
@@ -134,6 +138,41 @@ class Mongo {
       throw new Error('User not found');
     }
   }
+
+  async createSession(userId: string, ip: string, userAgent: string): Promise<SessionType> {
+    const ret = await this.sessionCollection.insertOne({
+      userId,
```

Comment:
"It is good to note that between the ObjectId (http://docs.mongodb.org/manual/reference/object-id/) and it's hex representation there is in fact 12 bytes of difference, the ObjectId being 12 bytes and it's hex representation being 24."
http://stackoverflow.com/questions/27896979/difference-between-storing-an-objectid-and-its-string-form-in-mongodb

It take less size in db and allow to have better performance on indexes.